pkg/apiserver/proxy.go

```diff
@@ -55,7 +56,7 @@ func (r *ProxyHandler) ServeHTTP(w http.ResponseWriter, req *http.Request) {
 	var apiResource string
 	var httpCode int
 	reqStart := time.Now()
-	defer monitor("proxy", &verb, &apiResource, &httpCode, reqStart)
+	defer monitor("proxy", verb, apiResource, util.GetClient(req), httpCode, reqStart)
```

Comment:
Switching verb and apiResource from pointers to actual strings will cause them to always be empty, because the arguments to a deferred function are evaluated at the point of the deferral.

See the [docs](http://blog.golang.org/defer-panic-and-recover) for details:
"1. A deferred function's arguments are evaluated when the defer statement is evaluated."